pyproject.toml

```diff
@@ -19,6 +19,7 @@ dependencies = [
   "requests==2.31.0",
   "requirements-parser==0.9.0",
   "setuptools==69.5.1",
+  'snowflake.core==0.8.0; python_version < "3.12"',
```

Comment:
Why python limit?

Reply:
The library doesn't work with 3.12 yet - it caused the build errors

Reply:
This means cli won't work with 3.12, right? Should we limit the cli? This is possibly breaking change 🤔